src/mirco_setparameters.cpp

```diff
@@ -15,7 +15,8 @@ void MIRCO::SetParameters(double& E1, double& E2, double& LateralLength, double&
     double& ElasticComplianceCorrection, double& GridSize, double& Tolerance, double& Delta,
     std::string& TopologyFilePath, int& Resolution, double& InitialTopologyStdDeviation,
     const std::string& inputFileName, bool& RandomTopologyFlag, double& Hurst, bool& RandomSeedFlag,
-    int& RandomGeneratorSeed, bool& WarmStartingFlag, int& MaxIteration, bool& PressureGreenFunFlag)
+    int& RandomGeneratorSeed, bool& WarmStartingFlag, int& MaxIteration, bool& PressureGreenFunFlag,
+    double& ExpectedPressure, double& ExpectedPressureTolerance)
```

Comment:
This is getting quite long. This should be replaced with a parameter container in the future, to improve readability and usability.